backend/app/db/crud/document.py

```diff
@@ -370,6 +371,7 @@ def get_document_detail(db, import_id_or_slug) -> DocumentDetailResponse:
     return DocumentDetailResponse(
         id=document_id,
         name=cast(str, document.name),
+        postfix=cast(str, document.postfix),
```

Comment:
Looking at this, I think we need to change `postfix` & `source_url` to be `Optional[str]` here, or we're likely to have a bad time. The old cast of `source_url` is from a naive time when we thought all documents required a source :(